Kentico.Kontent.Delivery.Tests/Extensions/ServiceCollectionsExtensionsTests.cs

```diff
@@ -63,92 +50,84 @@ public class ServiceCollectionsExtensionsTests
         private readonly ReadOnlyDictionary<Type, Type> _expectedInterfacesWithImplementationFactoryTypes = new ReadOnlyDictionary<Type, Type>(
            new Dictionary<Type, Type>
            {
-               { typeof(IContentLinkUrlResolver), typeof(DefaultContentLinkUrlResolver) },
-               { typeof(IContentLinkResolver), typeof(ContentLinkResolver) },
-               { typeof(ITypeProvider), typeof(TypeProvider) },
-               { typeof(IDeliveryHttpClient), typeof(DeliveryHttpClient) },
-               { typeof(IInlineContentItemsProcessor), typeof(InlineContentItemsProcessor) },
-               { typeof(IInlineContentItemsResolver<object>), typeof(ReplaceWithWarningAboutRegistrationResolver) },
-               { typeof(IInlineContentItemsResolver<UnretrievedContentItem>), typeof(ReplaceWithWarningAboutUnretrievedItemResolver) },
-               { typeof(IInlineContentItemsResolver<UnknownContentItem>), typeof(ReplaceWithWarningAboutUnknownItemResolver) },
-               { typeof(IModelProvider), typeof(ModelProvider) },
-               { typeof(IPropertyMapper), typeof(PropertyMapper) },
-               { typeof(IRetryPolicyProvider), typeof(DefaultRetryPolicyProvider) },
-               { typeof(IDeliveryClient), typeof(DeliveryClient) },
-               { typeof(IDeliveryClientFactory), typeof(DeliveryClientFactory) },
+                { typeof(IDeliveryClientFactory), typeof(DeliveryClientFactory) },
            }
-   );
+       );
+
+        public static IEnumerable<object[]> DeliveryOptionsConfigurationParameters =>
+           new[]
+           {
+                new[] {"as_root"},
+                new[] {"under_default_key", "DeliveryOptions"},
+                new[] {"under_custom_key", "CustomNameForDeliveryOptions"},
+                new[] {"nested_under_default_key", "Options:DeliveryOptions"}
+           };
 
-        private readonly FakeServiceCollection _fakeServiceCollection;
```

Comment:
the same question as above - why did we have to remove this, and consequently the tests?

Reply:
it seems that we're missing quite a lot of test scenarios. are we going to cover them somehow?